flow/util/misc.py

```diff
@@ -163,11 +163,11 @@ def __hash__(self):
         return hash(tuple(sorted(self.items())))
 
 
-def to_hashable(l):
+def to_hashable(line):
```

Comment:
Why did you name this argument `line`?

Reply:
There's no specific reason, I was waiting for the reviews.

Reply:
I assume that flake8 is complaining because a single letter variable, but it won't fail commit hooks unless you actually try to commit a change in the line in the code where the error is.

@kidrahahjo when you change a variable name like this, it's important to give it a name that is meaningful, not just something that prevents flake8 from throwing errors. In this case, I would suggest something generic but still descriptive like `obj` since the input is an object that *may* be a list, a dict, or something else.

Reply:
@vyasr okay, I'll keep that in mind.